code/modules/surgery/dental_implant.dm

```diff
@@ -41,3 +59,24 @@
 		target.reagents.trans_to(owner, target.reagents.total_volume, transfered_by = owner, method = INGEST)
 	qdel(target)
 	return TRUE
+
+/datum/action/item_action/hands_free/activate_signaler
+	name = "Activate Signaling Device"
+
+/datum/action/item_action/hands_free/activate_signaler/Trigger()
+	if(!..())
+		return FALSE
```

Comment:
```suggestion
/datum/action/item_action/hands_free/activate_signaler/Trigger()
	. = ..()
	if(!.)
		return
```

Reply:
Yeah this stuff with the dots is a little bit gibberish to me, still learning.